pye3d/detector_3d.py

```diff
@@ -334,7 +334,7 @@ def _predict_from_3d_search(
             _EYE_RADIUS_DEFAULT,
             self.camera.focal_length,
             self.camera.resolution,
-            major_axis_factor=2.5,
```

Comment:
I feel like this should be in a separate commit. What does it change?

Reply:
Yes, true. This sets the size of the crop for the 3d_search. Had forgotten that I had changed this for testing purposes.